examples/ReorderExample.js

```diff
@@ -94,6 +94,7 @@ class ReorderExample extends React.Component {
             columnKey={columnKey}
             key={i}
             isReorderable={true}
+            pureRendering={true}
```

Comment:
Ya let's remove it.  I'd rather avoid getting issues filed against it until it's stabilized a bit.